pages/index.js

```diff
@@ -1,8 +1,41 @@
 import ProductsList from "@/components/ProductsList";
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledCreateButton = styled.div`
+  display: inline-block;
+  background-color: var(--accentColor);
+  position: fixed;
+  bottom: 50px;
+  left: 50%;
+  transform: translate(-50%);
+  height: 60px;
+  width: 60px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50px;
+  box-shadow: 0px 1px 3px var(--primaryFontColor);
+  &:hover {
+    bottom: 47.5px;
+    right: 47.5px;
+    height: 65px;
+    width: 65px;
+  }
+`;
+
+const StyledLink = styled(Link)`
+  font-size: 1.4rem;
+  color: white;
+`;
+
 export default function HomePage({ products }) {
   return (
     <main>
       <ProductsList products={products}></ProductsList>
+      <StyledLink href="/create">
+        <StyledCreateButton>+</StyledCreateButton>
```

Comment:
I don't think we need the `div` here. We can move the styles to Link component and achieve the same affect.

Reply:
We might then want to consider 

`text-decoration: none`

Reply:
True! We removed the div :)

Reply:
Hello There.

Looks like you made the suggested changes. Looks good to us! Great job.

![borat](https://github.com/GoekhanAgkurt/capstone-grocery-app/assets/8994153/1690ca03-1df8-4c4a-871e-38ebc12908f1)